code/modules/surgery/bodyparts/bodypart_dismemberment.dm

```diff
@@ -357,7 +357,7 @@
 	if(brain)
 		if(brainmob)
 			brainmob.forceMove(brain) //Throw mob into brain.
-			brain.brainmob = brainmob //Set the brain to use the brainmob
+			//brain.brainmob = brainmob //Set the brain to use the brainmob
```

Comment:
I think..this needs to stay..or bad things happen when people get decappied and leave the body..just a thought..i dunno i am back tracking some stuff for a bug. SOMETHING is deleted the mind datum on a decap earlier. JOIN ME IN FINDING OFFENDING CODE ACROSS PRS